services/dashboard/views/apps/compose/popups/choose_edge.html

```diff
@@ -1,5 +1,5 @@
 <div class="modal" tabindex="-1">
-  <div class="modal-dialog modal-md">
+  <div class="modal-md col-md-5 col-md-offset-3">
```

Comment:
`modal-md` is my invented style, You still need `modal` as base style. Don't remove that.